vaadin-login-flow-parent/vaadin-login-testbench/src/main/java/com/vaadin/flow/component/login/testbench/LoginOverlayElement.java

```diff
@@ -53,27 +47,33 @@ public boolean isOpened() {
 
     @Override
     public TextFieldElement getUsernameField() {
-        return getLoginForm().getUsernameField();
+        return $(TextFieldElement.class).id("vaadinLoginUsername");
     }
 
     @Override
     public PasswordFieldElement getPasswordField() {
-        return getLoginForm().getPasswordField();
+        return $(PasswordFieldElement.class).id("vaadinLoginPassword");
     }
 
     @Override
     public ButtonElement getSubmitButton() {
-        return getLoginForm().getSubmitButton();
+        return $(ButtonElement.class).first();
```

Comment:
I'd suggest to query for the submit slot here, similar to how it works for forgot password below.

Reply:
Thanks, done.